src/pages/NewList.js

```diff
@@ -9,16 +9,11 @@ A shopping list consists of a set of items associated with a user’s token. Tok
   [x] Generate a new, unique token
   [x] Save the token to localStorage
   [x] Show the user the list view
-
-The following script can be used to generate a suitable token: https://gist.github.com/segdeha/21a42618ce5a54916c5b58d36ec2992e
 */
 
 const NewList = () => {
-  const generatedToken = getToken();
```

Comment:
This line wasn't needed, so I removed it. I made `initialToken` a function so that `getToken()` will only run once if needed when `token` is initialized.